File: ASPNETCore/HowTo/WebApiTypescriptSample/src/Modules/Excel/Dtos/GenerateExcelFromJSON.dto.ts
import { BaseDto } from "../../../Base/Base.Dto";

export class GenerateExcelFromJSONDto extends BaseDto {

    /**The exported file name. */
    public FileName: string;

    /**The Template file name (if not provide, use generated template automatically). */
    public TemplateFileName: string;

    /**The file format of the excel. */
    public Type: string;

    /**A data collection which is posted from client side. */
    public Data: string;
    
}

File: ASPNETCore/HowTo/WebApiConsoleSample/src/Modules/Excel/Dtos/GenerateExcelFromJSON.dto.cs
namespace WebApiConsoleSample
{
    public class GenerateExcelFromJSONDto : BaseDto
    {
        /**The exported file name. */
        public string FileName { set; get; }
        /**The Template file name (if not provide, use generated template automatically). */
        public string TemplateFileName { set; get; }
        /**The file format of the excel. */
        public string Type { set; get; }
        /**A data collection which is posted from client side. */
        public string Data { set; get; }

    }
}